### PR TITLE
MH-12885 capture agent access management documentation

### PR DESCRIPTION
**Pull request by staubesv**

### CODE REVIEW

File: docs/guides/user/docs/advanced/capture-agent-access.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Capture Agent Access Control
+
+Opencast allows restrictions considering what a user or group of users can do with capture agents to be configured.
+
+There are three kind of access restriction levels:
```
Comment (pmiddend) [inline docs/guides/user/docs/advanced/capture-agent-access.md]: *kinds

File: docs/guides/user/docs/advanced/capture-agent-access.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+- Unrestriced Access: Full access to all capture agents
```
Comment (pmiddend) [inline docs/guides/user/docs/advanced/capture-agent-access.md]: *Unrestricted

File: docs/guides/user/docs/advanced/capture-agent-access.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- Edit workflow configuration of scheduled events (Events->Event Details->Processing is read-only)
+- Schedule new events (Events->Add Event->Source will only allow upload)
+- Delete scheduled events (Events->Actions->Delete and Events->Action->Delete)
+- Remove capture agents (Locations->Locations->Action->Remove)
+
+### Selective Access
+
+It is possible to selectively allow specific users or groups of users to access subsets of capture agents.
+
+Each capture agent has a role which is derived froms its id by removing all non-alphanumerical characters and
```
Comment (pmiddend) [inline docs/guides/user/docs/advanced/capture-agent-access.md]: *from its

File: docs/guides/user/docs/advanced/capture-agent-access.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+prepending the prefix `ROLE_CAPTURE_AGENT_`.
+If a user with restriced access owns the roles of a set of given capture agents, the user has selective access
```
Comment (pmiddend) [inline docs/guides/user/docs/advanced/capture-agent-access.md]: *restricted

File: docs/guides/user/docs/advanced/index.md [hunk 5]
```diff
@@ -2,4 +2,15 @@
 
 This section describes facilities supposed to be used by advanced users.
 
-- [Role-Based Visibility](role-based-visibility.md)
\ No newline at end of file
+### Unprivileged Users
+
+If a user owns the global administration role (`ROLE_ADMIN`), he is a privileged users in means of having access
```
Comment (pmiddend) [inline docs/guides/user/docs/advanced/index.md]: *user

Comment (pmiddend) [reply docs/guides/user/docs/advanced/index.md]: You can avoid the "he" by writing "If users own the global administrator role, they are privileged users", by the way.

**Comment by staubesv** [comment]
@pmiddend Fixed ;-)

**Pull request merged**